src/util/permissionIdpGroups.tsx

```diff
@@ -21,3 +31,72 @@ export const testDuplicateIdpGroupName = (
     },
   ];
 };
+
+type DeleteIdpGroupsProps = {
+  idpGroups: IdpGroup[];
+  onSuccess?: () => void;
+};
+
+export const useDeleteIdpGroups = ({
+  idpGroups,
+  onSuccess,
+}: DeleteIdpGroupsProps) => {
```

Comment:
```suggestion
export const useDeleteIdpGroups = (idpGroups: IdpGroup[]) => {
```
The on success is never used. Make this a simpler interface with passing a single array, and remove the custom type.

Reply:
aah nice spot :+1: